### DISCUSSION TITLE
Who's Hiring (Jan 2024)

### DISCUSSION THREAD

**Discussion by kamaalsultan** [vercel/next.js#60667] [Show and tell]
# What is this?
 This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.
 
 # Companies
 Please use the following format followed by a brief description of the role:
 
 > COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK
 
 Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.
 
 Good luck with recruiting!
 
 ## Sample Job Post
 Here's a good example job post.
 
 Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
 Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
 > I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.
 
 If you hire someone through this job board, please reach out to me ([hassan@vercel.com](mailto:hassan@vercel.com)) and let me know so I can send you some swag :)
 
 If you're a developer looking to post about themselves, use #55075.
 
 We've had a lot of [success stories](https://twitter.com/nutlope/status/1691475914469904384) of people hiring from our job board!

**Comment by bitttttten** [comment]
Creative Fabrica | Senior Front-end Developer | Amsterdam | Full-time

We're Creative Fabrica, a fast scaling tech start-up from Amsterdam on the look out for experienced Front-end Developers to work with us creating new AI tools or crafts marketplace platforms with TypeScript, Next.js, turborepo, TailwindCSS, radix ui all with our GraphQL and Golang backend.

You'll be joining small teams where we have a great building culture. We have a lot going on, so you can be a part of the AI tools, design tools, or our large crafting marketplace (think almost 50m+ visitors a month) with many greenfield projects. Our Generative AI models are trained in house and our AI tools are built in house too! If you're excited about working on new projects and having a real impact in their development then this is the role for you.

Currently we're only looking for in office positions, so no remote options yet.

TLDR:
🇳🇱 Office in Amsterdam with lunch provided and Friday drinks
👩‍💻 React, Next.js, Turborepo, Tailwind, Vitest, vite, Radix UI, Vercel, AWS, CloudFlare
🤖 Work with unique, first to market, AI tools and a popular crafters marketplace

Any other questions let me know! 

https://grnh.se/eeb72142teu

**Reply by yasinkavakli** [reply]
Sadly not remote, would've been pretty interesting project.
I'm from Austria and bought a house 3 years ago and have 2 little kids, so relocation is not an option.

But still, good luck to anyone applying 🙌

**Reply by arielconti10** [reply]
Do you sponsor relocation across Europe?
